src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -72,6 +72,112 @@ public void VerifyMSTestTemplate(DotNetLanguage language)
             DotNetSdkActions.Test);
         newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
     }
+    
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    public void VerifyCSharpWPFTemplate(DotNetLanguage language)
```

Comment:
The two tests here can be collapsed into one, with multiple "InlineData" attributes. Xunit will then run the test method for each of the input data items.

Reply:
This has been fixed with multiple InlineData attributes